QgisModelBaker/gui/topping_wizard/referencedata_page.py

```diff
@@ -145,7 +145,7 @@ def update_referecedata_cache_model(self, filter_models, type):
         timer = QTimer()
         timer.setSingleShot(True)
         timer.timeout.connect(lambda: loop.quit())
-        timer.start(10000)
+        timer.start(3000)
```

Comment:
Here we have 3 seconds to be supersure and there the next-page does not rely on this timeout (compared to the one in the workflow wizard)...